lib/postgresql.js

```diff
@@ -119,7 +119,12 @@ PostgreSQL.prototype.executeSQL = function (sql, params, callback) {
       // console.log(err);
       if (log) log(sql, time);
       process.nextTick(done); // Release the pooled client
-      callback(err ? err : null, data ? data.rows : null);
+
+      var affectedCount = data && typeof data.rowCount === 'number' &&
+          !isNaN(data.rowCount) ? data.rowCount : 0;
+
+      callback(err ? err : null, data ? data.rows : null, {count:
+          affectedCount});
```

Comment:
+1 to callback with the full `data` object and remove the 3rd arg.

We should use `undefined` or `-1` if the count is unknown.